lib/tree-iterator.js

```diff
@@ -54,7 +54,12 @@ function iterate(node, cb) {
                 XJSOpeningElement: ['name', 'attributes'],
                 XJSAttribute: ['name', 'value'],
                 XJSSpreadAttribute: ['argument'],
-                XJSText: null
+                XJSText: null,
+                // Esprima 2.3 introduced handlers and handler which duplicated
+                // CatchClause nodes. Can't update estraverse until we
+                // replace esprima-harmony-jscs for babel
+                // See https://github.com/jquery/esprima/issues/1031
+                TryStatement: ['block', 'handler', 'finalizer']
```

Comment:
Weird that you didn't need to update tests because handlers was an array where as handler isn't. 

Can you please double check? Also, wondering if we should backCompat this until 2.0?